gnd/src/main/java/com/google/android/gnd/ui/offlinearea/OfflineAreaManagerViewModel.java

```diff
@@ -0,0 +1,12 @@
+package com.google.android.gnd.ui.offlinearea;
+
+import com.google.android.gnd.ui.common.AbstractViewModel;
+
+import javax.inject.Inject;
+
+public class OfflineAreaManagerViewModel extends AbstractViewModel {
```

Comment:
Please add minimal class JavaDoc (e.g., "View state and logic for UI for managing areas to be cached for offline use.").